react/features/conference/components/Conference.native.js

```diff
@@ -190,7 +190,6 @@ class Conference extends Component<Props> {
             <Container
                 accessibilityLabel = 'Conference'
                 accessible = { false }
-                onClick = { this._onClick }
```

Comment:
1. Added explanations in VideoTransform#_onGesture and ParticipantView#render to explain these changes. Please review them, I'm happy to extend them if need be.
2. In order to extract that in a separate commit and not to loose functionality I'd need to implement a temporary solution to detect touches over the video (see previous explanations) without having the VideoTransform component in place (yet) - which I'd have to delete immediately when adding the VideoTransform component in the next commit. I'm happy to do this, but is it worth the effort?

Reply:
> In order to extract that in a separate commit and not to loose functionality I'd need to implement a temporary solution to detect touches over the video (see previous explanations) without having the VideoTransform component in place (yet) - which I'd have to delete immediately when adding the VideoTransform component in the next commit. I'm happy to do this, but is it worth the effort?

You've already developed a "clickable avatar" in this PR. Which doesn't really have to be an "avatar" and may instead be a higher-order "clickable". At the end of the day you've already identified two "views" which we may want to add "clickable" capabilities to so I'd say the chances are we'll find more. And with one stone we'd have hit a second bird: the unreadably formatted "clickable avatar" implemented in `ParticipantView`.

Reply:
Here's an initial `Pressable` React `Component` which I wrote in react/features/base/react/components/native/:

```javascript
// @flow

import React, { Component } from 'react';
import { TouchableWithoutFeedback } from 'react-native';

/**
 * The type of the React {@link Component} props of {@link Pressable}.
 */
type Props = {
    children: React$Node,

    /**
     * Called when the touch is released, but not if cancelled (e.g. by a scroll
     * that steals the responder lock).
     */
    onPress: Function
};

/**
 * Adds support for {@code onPress} to a child React {@link Component} (which
 * should probably not support the prop in question; otherwise, there's little
 * point of using {@code Pressable} then in the first place).
 */
export default class Pressable extends Component<Props> {
    /**
     * Implements React's {@link Component#render()}.
     *
     * @inheritdoc
     * @returns {React$Node}
     */
    render() {
        // onPress
        const { children, onPress } = this.props;

        if (onPress) {
            return (
                <TouchableWithoutFeedback onPress = { onPress }>
                    { children }
                </TouchableWithoutFeedback>
            );
        }

        // A Pressable without an onPress is a "no-op".
        return children;
    }
}
```

Reply:
@zbettenbuk, I've demonstrated the separation of the `onPress` modifications which prepare for "pinch-to-zoom" from the `VideoTransform`-related source code by implementing it. Let's please try to finish this PR by keeping the commits atomic (even if the next requires the previous).